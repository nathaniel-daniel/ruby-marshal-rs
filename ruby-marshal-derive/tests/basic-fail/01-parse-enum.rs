#[derive(ruby_marshal_derive::FromValue)]
pub enum MyObject {
    A {
        field1: i32,
    },
    B {
        field2: i32,
    },
}

fn main() {}