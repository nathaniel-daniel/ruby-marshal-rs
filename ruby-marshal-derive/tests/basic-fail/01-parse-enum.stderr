error: only structs are supported
 --> tests/basic-fail/01-parse-enum.rs:2:1
  |
2 | / pub enum MyObject {
3 | |     A {
4 | |         field1: i32,
5 | |     },
... |
8 | |     },
9 | | }
  | |_^
